tools/src/main/java/org/apache/kafka/trogdor/coordinator/NodeManager.java

```diff
@@ -192,6 +191,9 @@ public void run() {
                     // agents going down?
                     return;
                 }
+                if (log.isTraceEnabled()) {
```

Comment:
do we need this `if`, since we are not doing any calculations for arguments we pass to `log.trace`?

Reply:
`agentStatus#toString` is pretty heavy.  If you have a dozen workers going on, it will serialize information about all of them into a string.